### PR TITLE
Allow hardcoding numel and use CPUAdam

### PR DESCRIPTION
**Pull request by ver217**

### CODE REVIEW

File: zero/colossalai_utils/gpt2_config.json [hunk 1]
```diff
@@ -1,11 +1,12 @@
 {
     "method": "colossalai",
     "model": {
-        "type": "gpt2_10b"
+        "type": "gpt2_10b",
+        "numel": 9.57
```
Comment (feifeibear) [inline zero/colossalai_utils/gpt2_config.json:5]: ? why hard code?

**Review by FrankLeeeee — APPROVED** [review]

File: zero/colossalai_utils/gpt2_config.json (hunk 1 above)
Comment (FrankLeeeee) [reply zero/colossalai_utils/gpt2_config.json:5]: It takes too long to build the model. We want to save some time for now.

Comment (ver217) [reply zero/colossalai_utils/gpt2_config.json:5]: Initialization time is too long.

Comment (ver217) [reply zero/colossalai_utils/gpt2_config.json:5]: This is suitable for all methods (Fairscale / Deepspeed / Ours).

**Pull request merged**